plugins/scaffolder-backend/src/scaffolder/actions/builtin/publish/github.ts

```diff
@@ -67,6 +68,10 @@ export function createPublishGithubAction(options: {
             type: 'string',
             enum: ['private', 'public', 'internal'],
           },
+          sourcePath: {
+            title: 'Repository Path',
```

Comment:
Let's get a bit more descriptive here :grin:

"Path within the workspace that will be used as the repository root. If omitted the entire workspace will be published as the respository."?